lib/build/recipe/session/index.js

```diff
@@ -221,7 +221,7 @@ class SessionWrapper {
                 options,
                 userContext,
             });
-            if (res.status === "OK" && res.session !== undefined) {
```

Comment:
- we need to create just one recipe function for createNewSession which doesn't take a req / res (but the request will be passed to it via userContext)
- The createNewSession(req, res, ..) will also call this function, and set the userContext correctly.
- The createNewSessionWithoutReqeustResponse will also call this same function, but will not have req in userContext